start_stop.lua

```diff
@@ -210,7 +210,15 @@ function courseplay:start(self)
 		self.cp.driver = UnloadableFieldworkAIDriver.create(self)
 	end
 	StartStopEvent:sendStartEvent(self)
+  -- Remeber crab steering if relevant
```

Comment:
So why is this code here, we already have a part dealing with crab steering a few lines above this (181)?

Reply:
I've never managed to get through the if clause on L181 even when having a crab capable vehicle.
The debugger didn't help as for some reason it detects 2 self variables and chooses to show only the other.
Finally, even if you did manipulate the mode up there, it's changed later by self.cp.driver:start calling some Giants code for the AI helper. You've looked on the subject some (a lot of) time ago at #3062, not sure if this helps.

Reply:
Ok, I see, thanks for the link. 

In that case however, the proper solution is to override the CrabSteering:onAIImplementStart() Giants function, like we do with some others in FieldworkAIDriver:register().

Reply:
I understand. I'll have to read through the source to understand how to deal with this.
Would this be similar to how you return false here?
https://github.com/Courseplay/courseplay/blob/93ec4a3feda28a8fe542abd35d7c6f6069a8b746/FieldworkAIDriver.lua#L98

Reply:
yeah, pretty much the same thing, just check the keepCurrentSetting in the new onAIImplementStart() and if it is true, don't call the setCrabSteering(), I believe that's all.